packages/dmn-js-drd/src/features/modeling/behavior/ReplaceConnectionBehavior.js

```diff
@@ -0,0 +1,40 @@
+import inherits from 'inherits';
+
+import CommandInterceptor from 'diagram-js/lib/command/CommandInterceptor';
+
+
+export default function ReplaceConnectionBehavior(injector, modeling, rules) {
+  injector.invoke(CommandInterceptor, this);
+
+  this.postExecute('connection.reconnect', function(context) {

```

Comment:
In bpmn-js we do it pre-execute, plus we carry out a number of additional activities, too, cf. [`ReplaceConnectionBehavior`](https://github.com/bpmn-io/bpmn-js/blob/develop/lib/features/modeling/behavior/ReplaceConnectionBehavior.js#L149).

Why should we do post execute here?

Reply:
No good reason.